armi/reactor/blocks.py

```diff
@@ -1663,6 +1663,10 @@ class HexBlock(Block):
     .. impl:: ARMI has the ability to create hex shaped blocks.
         :id: I_ARMI_BLOCK_HEX
         :implements: R_ARMI_BLOCK_HEX
+
+        This class defines hexagonal-shaped Blocks. It inherits functionality from the parent
+        class, Block, and defines hexagonal-specific methods including, but not limited to,
+        pin pitch, hydraulic diameter, and retrieving inner and outer pitch.
```

Comment:
Wondering if we should add updating block-level pin linear power densities here.  `setPinPowers`

Reply:
https://github.com/terrapower/armi/pull/1619/commits/c8a671365bff81b372d7b131a64cadeaacefaaf5